Doc/library/typing.rst

```diff
@@ -604,18 +604,16 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
    * Redundant arguments are skipped, e.g.::
 
-       Union[int, str, int] == Union[int, str]
+       Union[int, str, int] == Union[int, str] == int | str
```

Comment:
Not a critique and slightly off topic: this threw me off a little bit, because in earlier versions of 3.10 this was false because they are different types at runtime. Only since rc1 this is true.

Reply:
Interesting, I didn't know that.

Reply:
Sorry for the misinformation, but I was wrong 😉 . I think my memory is getting hazy, it was `hash(int | str)` that wasn't equivalent to `hash(Union[int, str])` up till rc1.